components/eamxx/cime_config/config_component.xml

```diff
@@ -27,12 +27,12 @@
     <valid_values></valid_values>
     <default_value></default_value>
     <values match='last'>
-      <value compset="_SCREAM">SCREAM_NP 4 SCREAM_NUM_VERTICAL_LEV 72 SCREAM_NUM_TRACERS 10</value>
-      <value compset="_SCREAM%ZM">SCREAM_NP 4 SCREAM_NUM_VERTICAL_LEV 72 SCREAM_NUM_TRACERS 10</value>
+      <value compset="_SCREAM">SCREAM_NP 4 SCREAM_NUM_VERTICAL_LEV 128 SCREAM_NUM_TRACERS 10</value>
+      <value compset="_SCREAM%ZM">SCREAM_NP 4 SCREAM_NUM_VERTICAL_LEV 128 SCREAM_NUM_TRACERS 10</value>
```

Comment:
I don't see a diff between the case `"_SCREAM"` and the case `"_SCREAM%ZM"`. @whannah1 unless we foresee ZM cases to need additional eamxx cmake options, maybe we can just rm this?

Reply:
yea, I guess that was leftover from the early phase of implementing the fortran bridge for ZM. I don't think we need additional cmake options as long as we control whether the ZM process is added or not.